extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts

```diff
@@ -129,6 +132,11 @@ export class RemoteQueriesInterfaceManager {
     });
   }
 
+  public async openFile(filePath: string) {
```

Comment:
Since this is only used inside the class, it can be made private.

```suggestion
  private async openFile(filePath: string) {
```